modules/services/ui/src/main/res/values/themes.xml

```diff
@@ -197,6 +197,7 @@
         <item name="isDark">false</item>
         <item name="isImageTintEnabled">false</item>
         <item name="imageTintThemeCacheTag">light</item>
+        <item name="enableEdgeToEdge">true</item>
```

Comment:
The `BottomSheetDialog` material component uses this to know it should draw to the edge which fixes the navigation bar being a different colour to the background.